docsets/Celery/docset.json

```diff
@@ -1,14 +1,18 @@
 {
     "name": "Celery",
-    "version": "3.1.23",
+    "version": "4.1.0",
     "archive": "Celery.tgz",
     "author": {
-        "name": "Cameron Will",
-        "link": "https://github.com/cwill747"
+        "name": "James M. Allen",
+        "link": "https://github.com/jamesmallen"
     },
     "aliases": [],
 
     "specific_versions": [
+        {
+            "version": "4.1.0",
+            "archive": "versions/4.1.0/Celery.tgz"
+        }
```

Comment:
The JSON isn't valid. I think there's a missing comma at the end of this line.

Reply:
Woops - fixed!